src/test/scala/org/camunda/feel/impl/interpreter/InterpreterUnaryTest.scala

```diff
@@ -147,6 +147,8 @@ class InterpreterUnaryTest
     evalUnaryTests("c", """ "a","b" """) should be(ValBoolean(false))
   }
 
+  /*
+  TODO: These tests currently fail, but the specified behavior seems rather surprising anyway
```

Comment:
These tests are correct and must be stay enabled. 

Example use case:
* a decision table with an input that is named "is valid"
* the input has an input expression "contract.isValid"
* a rule with an input entry with the unary-test `true`
* the rule should only match if `contract.isValid` is equal to `true`